### PR TITLE
Feature/7/application layout

### PR DESCRIPTION
**Pull request by Kosik33i6**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
 import React from "react";
+import MainLayout from "./components/layout/MainLayout/MainLayout";
+import Navigation from "./components/layout/Navigation/Navigation";
+import Header from "./components/layout/Header/Header";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
```
Comment (m-ossolinski) [inline src/App.js:5]: reorder this import and move it to top

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
 
 const App = () => {
   return (
     <>
-      <h1>Welcome to Coders Camp 2021</h1>
-      <h2>SPA Project</h2>
+      <BrowserRouter>
+        <Navigation />
+        <MainLayout>
+          <Routes>
```
Comment (m-ossolinski) [inline src/App.js:13]: add separatate folder under `/src` for routes (`/src/routes`) and create routes agregator there

File: src/components/layout/MainLayout/MainLayout.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import PropTypes from "prop-types";
+
+const MainLayout = ({ children }) => {
+  return (
+    <div>
```
Comment (m-ossolinski) [inline src/components/layout/MainLayout/MainLayout.js:5]: redundant div

File: src/components/layout/Navigation/Navigation.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      <nav>
+        <Link to="/home">Home</Link>
+        <Link to="/transactions">Transactions</Link>
+        <Link to="/signup">Signup</Link>
+      </nav>
+    </div>
+  );
+};
+
+export default Navigation;
```
Comment (m-ossolinski) [inline src/components/layout/Navigation/Navigation.js:18]: remove export default

File: src/components/layout/MainLayout/MainLayout.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+      <main className="main">{children}</main>
+    </div>
+  );
+};
+
+MainLayout.propTypes = {
+  children: PropTypes.node,
+};
+
+export default MainLayout;
```
Comment (m-ossolinski) [inline src/components/layout/MainLayout/MainLayout.js:15]: remove export default

File: src/components/layout/Header/Header.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+const Header = () => {
+  return (
+    <header className="header">
+      <h1>Welcome to Coders Camp 2021</h1>
+      <h2>SPA Project</h2>
+    </header>
+  );
+};
+
+export default Header;
```
Comment (m-ossolinski) [inline src/components/layout/Header/Header.js:10]: remove export default

**Review by m-ossolinski — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 7]
```diff
@@ -1,12 +1,23 @@
...
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  margin: 0 auto;
+  padding: 2rem;
+`;
 
 const App = () => {
   return (
     <>
```
Comment (m-ossolinski) [inline src/App.js:16]: reduntant

**Review by m-ossolinski — APPROVED** [review]

**Review by zakrzaq — APPROVED** [review]

**Pull request merged**